awslimitchecker/tests/test_runner.py

```diff
@@ -614,6 +619,19 @@ def test_entry_no_service_name(self, capsys):
         assert excinfo.value.code == 6
         assert self.cls.service_name is None
 
+    def test_entry_no_service_name_skip_service(self, capsys):
```

Comment:
Ok, well then -- thank you for the codereview, and suggested edits, they were all spot on.
I've definitely learned a few things along the way here.

I'll watch what you do to improve this test from the sidelines.